src/Provider/Keycloak.php

```diff
@@ -157,6 +263,15 @@ protected function getDefaultScopes()
         return ['name', 'email'];
     }
 
+    protected function getAuthorizationHeaders($token = null)
```

Comment:
This method is not needed. `use BearerAuthorizationTrait;` should be taking care of this already.